includes/html/print-map.inc.php

```diff
@@ -15,6 +15,7 @@
 use LibreNMS\Config;
 
 $highlight_node = $vars['highlight_node'] | 0;
+$group = $vars['group'] | 0;
```

Comment:
bitwise operators? are you sure?

Reply:
copied from line above :-)
can change both to a `$var = $arg ?: 0;`